src/main/groovy/org/ajoberstar/grgit/Commit.groovy

```diff
@@ -16,6 +16,9 @@
 package org.ajoberstar.grgit
 
 import groovy.transform.Immutable
+import org.ajoberstar.grgit.operation.AddOp
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The [org.ajoberstar.grgit.operation.AddOp] import is never referenced [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.ajoberstar.com/coding_rules#rule_key=grvy%3Aorg.codenarc.rule.imports.UnusedImportRule)